src/app/housing/effects/housing.spec.ts

```diff
@@ -77,34 +105,92 @@ describe('HousingEffects', () => {
     expect(executor).to.exist;
   });
 
-  it('will return same search action', fakeAsync(() => {
-    (mockHouseService.findHouses as sinon.SinonStub).returns(new BehaviorSubject([]));
+  // -- HousingAction.SEARCH --
+
+  it('will return a different action (from SEARCH to LIST_HOUSES)', fakeAsync(() => {
+    (mockHouseService.findHouses as sinon.SinonStub).returns(of<House[]>([]));
     executor.queue({ type: HousingAction.SEARCH });
     effect.search$.subscribe(result => {
-      expect(result.type).to.equal('House - list houses');
+      expect(result.type).to.equal(HousingAction.LIST_HOUSES.toString());
     });
   }));
 
   it('will return an empty search result by calling house service', fakeAsync(() => {
-    (mockHouseService.findHouses as sinon.SinonStub).returns(new BehaviorSubject([]));
+    (mockHouseService.findHouses as sinon.SinonStub).returns(of<House[]>([]));
     executor.queue(ActionFactory.search('Melbourne'));
     effect.search$.subscribe(result => {
       expect(result.payload.length).to.equal(0);
     });
   }));
 
   it('will return a filled result', fakeAsync(() => {
-    (mockHouseService.findHouses as sinon.SinonStub).returns(new BehaviorSubject(mockResponse));
+    (mockHouseService.findHouses as sinon.SinonStub).returns(of(mockResponse));
     executor.queue(ActionFactory.search('Sydney'));
     effect.search$.subscribe(result => {
       expect(result.payload).to.eql(mockResponse);
     });
   }));
 
   it('will return empty result when no search term provided', fakeAsync(() => {
+    const toastMessage = 'Reseting search results.';
+
     executor.queue(ActionFactory.search(''));
     effect.search$.subscribe((result) => {
       expect(result.payload.length).to.equal(0);
+      expect((mockToaster.info as sinon.SinonStub).calledWithExactly(toastMessage)).to.be.true;
+    });
+  }));
+
+  it('will gracefully handle an error that occurred in houseService.findHouses', fakeAsync(() => {
+    (mockHouseService.findHouses as sinon.SinonStub).returns(_throw('ka-BOOM!'));
+    const toastMessage = 'Something went horribly wrong while searching for "let\'s break things!".';
+
+    /* tslint:disable-next-line:quotemark */
+    executor.queue(ActionFactory.search("let's break things!"));
+    effect.search$.subscribe((result) => {
+      expect(result.payload.length).to.equal(0);
+      expect((mockToaster.error as sinon.SinonStub).calledWithExactly(toastMessage)).to.be.true;
+    });
+  }));
+
+  // -- HousingAction.ADD_HOUSE --
```

Comment:
Rather than these comments, maybe put them in a describe. 

that way the tests are nested and the comment get value?